src/hooks/use-global-store.ts

```diff
@@ -1,13 +1,22 @@
+import { AnyCircuitElement } from "circuit-json"
 import { create } from "zustand"
 import { persist } from "zustand/middleware"
 
+interface snippetInfo {
+  circuitJson?: AnyCircuitElement[] | null
+  code: string
+  snippetType: "board" | "package" | "model" | "footprint" | string | undefined
```

Comment:
The snippet type shouldn't be `undefined` or `string`, when we are using the const names. It defeats the point of it